src/Symfony/Component/Form/Extension/Core/DataTransformer/DateTimeToStringTransformer.php

```diff
@@ -110,7 +110,7 @@ public function reverseTransform(mixed $value): ?\DateTime
         }
 
         if (str_contains($value, "\0")) {
-            throw new TransformationFailedException('Null bytes not allowed');
+            throw new TransformationFailedException('Null bytes not allowed.');
```

Comment:
Revert?

Reply:
I've just applied patch that fabpot suggested